src/api/events/EventsApi.ts

```diff
@@ -35,7 +35,8 @@ export class EventsApi {
      * @summary Create Events
      * @param body
     */
-    public async createEvents(body: CreateEventsRequest, options?: FSRequestOptions): Promise<FSResponse<void>> {
+    public async createEvents(request: { body: CreateEventsRequest,  }): Promise<FSResponse<void>> {
```

Comment:
Maybe overthinking, but this could be changed from `body` to `payload` with a bit more changes in the template.